runtimes/aspnetcore-1.0/cloudbuild.yaml

```diff
@@ -35,6 +35,11 @@ steps:
           '--no-cache', '--pull', './versions/1.0.8' ]
   id: 'aspnetcore-1-0-8-build'
   waitFor: ['-']
+- name: gcr.io/cloud-builders/docker
```

Comment:
Is any of this done via a script? or all by hand?

Reply:
I do it by hand, but I have been thinking about doing some sort of generator.